recipe/meta.yaml

```
@@ -10,8 +11,16 @@ source:
   sha256: 9e8a7640911481134e60987d5d1e1cd157f430c3b4b38de8d36fc55c302bc299
 
 build:
-  number: 1
-  script: {{ PYTHON }} -m pip install .
+  number: {{ build_num }}
+  string: {{ impl }}_py{{ PY_VER }}h{{ PKG_HASH }}_{{ build_num }}
```

Shouldn't this also work? `conda-build` normally evaluates it after reading `number`
```suggestion
  string: {{ impl }}_py{{ PY_VER }}h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}
```